### PR TITLE
A few minor changes

### PR DESCRIPTION
**Pull request by devMichaelJones**
Please have a look at these changes I've accumulated while working with CppUTest.  Thanks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ TEST(FirstTestGroup, FirstTest)
 ## Command line switches
 
 * -v verbose, print each test name as it runs
-* -r# repeat the tests some number of times, default is one, default is # is not specified is 2. This is handy if you are experiencing memory leaks related to statics and caches.
+* -r# repeat the tests some number of times, default is one, default if # is not specified is 2. This is handy if you are experiencing memory leaks. A second run that has no leaks indicates that someone
```
Comment (basvodde) [inline README.md]: Uhm, is it me or is this not a complete sentence?

File: build/ComponentMakefile [hunk 2]
```diff
@@ -210,7 +210,7 @@ gcov: test
 	$(CPPUTEST_HOME)/scripts/filterGcov.sh $(GCOV_OUTPUT) $(GCOV_ERROR) $(GCOV_REPORT) $(TEST_OUTPUT)
 	cat $(GCOV_REPORT)
 
-.PHONEY: format
+.PHONY: format
```
Comment (basvodde) [inline build/ComponentMakefile]: Good catch :)

**Comment by basvodde** [comment]
Hilarious pull request :) Thanks!

Auto-complete definitively doesn't help in discovering stupid mistakes. The TestOuput of the VistualStudio :) I'll merge it even though there was one sentence that didn't seem complete. But we can fix that later :)

File: README.md (hunk 1 above)
Comment (devMichaelJones) [inline README.md]: You're right.  Looks like it had been that way until you fixed it on 2015-05-01 with commit 8414625.  I had originally made my change here before that, so something must've gone wrong when I was rebasing.  Oops- sorry for undoing that.  I'll add the re-fix to my new "minor" branch.

**Comment by devMichaelJones** [comment]
I notice that the branch was merged with a merge commit rather than just fast-forwarded.  I'm familiar with git, but new with github- is there a reason that github did it this way, or was it your preference?

**Comment by basvodde** [comment]
Uhm, no. I just hit the merge button :)

**Pull request merged**